.github/workflows/ci-cd.yml

```diff
@@ -18,82 +17,60 @@ jobs:
         with:
           dotnet-version: 5.0.x
 
-      - name: Data gatherer
-        id: data_gatherer
-        shell: pwsh
-        run: |
-          # Get default branch
-          $repo = 'microsoft/OpenAPI.NET'
-          $defaultBranch = Invoke-RestMethod -Method GET -Uri https://api.github.com/repos/$repo | Select-Object -ExpandProperty default_branch
-          Write-Output "::set-output name=default_branch::$(echo $defaultBranch)"
-
-      - name: Conditionals handler
-        id: conditionals_handler
-        shell: pwsh
-        run: |
-          $defaultBranch = "${{ steps.data_gatherer.outputs.default_branch }}"
-          $githubRef = "${{ github.ref }}"
-          $isDefaultBranch = 'false'
-          if ( $githubRef -like "*$defaultBranch*" ) {
-            $isDefaultBranch = 'true'
-          }
-          Write-Output "::set-output name=is_default_branch::$(echo $isDefaultBranch)"
-
       - name: Checkout repository
         id: checkout_repo
         uses: actions/checkout@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           fetch-depth: 0
 
-      - if: steps.conditionals_handler.outputs.is_default_branch == 'true'
+      - if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         name: Bump GH tag
         id: tag_generator
         uses: mathieudutour/github-tag-action@v5.4
         with: 
           github_token: ${{ secrets.GITHUB_TOKEN }}
           default_bump: false
-          release_branches: ${{ steps.data_gatherer.outputs.default_branch }}
+          release_branches: master
```

Comment:
```suggestion
          release_branches: vnext
```

Reply:
@baywet I replaced this logic to point from `vnext` to `master` based on this comment https://github.com/microsoft/OpenAPI.NET/pull/595#discussion_r658865021 by @darrelmiller

Reply:
ok ignore my confused self then :)